src/views/templates/details/TemplatePageTitle.tsx

```diff
@@ -26,7 +30,7 @@ const TemplatePageTitle: React.FC<TemplatePageTitleTitleProps> = ({ template })
           <Button
             variant="link"
             isInline
-            onClick={() => onBreadcrumbClick(`/k8s/ns/${namespace}/${VirtualMachineModelRef}`)}
+            onClick={() => onBreadcrumbClick(`/k8s/${namespacePath}/${VirtualMachineModelRef}`)}
           >
             {t('Virtualization')}
```

Comment:
this will create wrong breadcrumbs:
should be:
Pod > pod details
here we have:
Workloads > Pod > pod details

@upalatucci  can you remove the "virtualization" link ?